internal/kafka/internal/workers/kafka_mgrs/provisioning_kafkas_mgr.go

```diff
@@ -69,6 +69,7 @@ func (k *ProvisioningKafkaManager) Reconcile() []error {
 			}
 		}
 		metrics.UpdateKafkaRequestsStatusSinceCreatedMetric(constants2.KafkaRequestStatusProvisioning, kafka.ID, kafka.ClusterID, time.Since(kafka.CreatedAt))
+		metrics.UpdateKafkaRequestsStatusSinceLastSetMetric(constants2.KafkaRequestStatusProvisioning, kafka.ID, kafka.ClusterID)
```

Comment:
For provisioning status, the metric has to be done only here https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/ee6c732df5c8a67e06a5c2c74e26705e85d2c253/internal/kafka/internal/services/kafka.go#L476